packages/sanity/src/structure/panes/document/timeline/timelineItem.tsx

```diff
@@ -67,59 +131,42 @@ export function TimelineItem({
   )
 
   return (
-    <Root
-      $selected={isSelected}
-      $disabled={!isSelectable}
-      data-testid="timeline-item-button"
-      data-chunk-id={chunk.id}
-      data-first={isFirst ? true : undefined}
-      data-last={isLast ? true : undefined}
-      data-ui="timelineItem"
-      mode={isSelected ? 'default' : 'bleed'}
-      onClick={handleClick}
-      padding={0}
-      radius={2}
-      tone={isSelected ? 'primary' : TIMELINE_ITEM_EVENT_TONE[chunk.type]}
```

Comment:
Just to double check, we're not breaking some functionality with removing this right? 😬

Reply:
I think it should not affect any existing functionality, all of this props received in the element were used for styling.
I'm keeping this ones given they are used for ui and testing. 
```typescript
   data-ui="timelineItem"
   data-testid="timeline-item-button"
   data-chunk-id={chunk.id}
```